packages/transaction-log/src/services/api.service.ts

```
@@ -49,10 +51,9 @@ export async function getChannelById(id: String): Promise<Channel> {
   }
 }
 
-export async function getTransactions(filters: {}): Promise<any[]> {
+export async function getTransactions(filters: {}): Promise<Transaction[]> {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)